### PR TITLE
Using lakeFS with EMR docs

### PR DESCRIPTION
**Pull request by itaiad200**

### CODE REVIEW

File: docs/using/emr.md [hunk 1]
```diff
@@ -7,4 +7,36 @@ nav_order: 6
 has_children: false
 ---
 
-# Using lakeFS with EMR
\ No newline at end of file
+# Using lakeFS with EMR
+
+EMR is AWS big data platform for processing data using open source tools such as Spark, Hive and more.
```
Comment (ozkatz) [inline docs/using/emr.md:12]: Please use the description from here (or possibly link to it): https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-what-is-emr.html

File: docs/using/emr.md [hunk 2]
```diff
@@ -7,4 +7,36 @@ nav_order: 6
...
+
+Once you have your EMR cluster up and running, you can configure it to use your lakeFS installation as the s3 gateway.
+
+
+## Configuration
+In order to configure EMR to work with lakeFS we will set the lakeFS credentials and endpoint in the appropriate fields.
```
Comment (ozkatz) [inline docs/using/emr.md:18]:
```suggestion
In order to configure Spark on EMR to work with lakeFS we will set the lakeFS credentials and endpoint in the appropriate fields.
```

**Review by ozkatz — CHANGES_REQUESTED** [review]
This is a guide for Spark on EMR - I'd add another example at least for Hive.
Additionally, please add a note: EMR will encourage users to use `s3://` with Spark as it will use EMR's proprietary driver. Users need to use `s3a://` for this guide to work.

**Comment by itaiad200** [comment]
@ozkatz 
Tested both configurations (on-creation and on-adding-step) on the supported applications and they both work.

File: docs/using/emr.md [hunk 3]
```diff
@@ -7,4 +7,124 @@ nav_order: 6
...
+
+## Configuration on cluster creation 
+Use the below configuration when creating the cluster. You may delete any app configuration which is irrelavant for your use-case.
+ ```json
+[
+  {
+    "Classification": "presto-connector-hive",
+    "Properties": {
+      "hive.s3.aws-secret-key": "$LAKEFS_SECRET_KEY",
+      "hive.s3.aws-access-key": "$LAKEFS_SECRET_ACCESS_KEY",
```
Comment (ozkatz) [inline docs/using/emr.md:43]: I think the variables are reversed (i.e. `$LAKEFS_SECRET_ACCESS_KEY` should probably be `hive.s3.aws-secret-key`?)

File: docs/using/emr.md [hunk 4]
```diff
@@ -7,4 +7,124 @@ nav_order: 6
...
+
+## Configuration on adding a step
+When a cluster was created without the above configuration, you can still use lakeFS when adding a step.
+
+For example, when creating a Spark job: 
+
+```shell
+aws emr add-steps --cluster-id j-197B3AEGQ9XE4 \
+--steps="Type=Spark,Name=SparkApplication,ActionOnFailure=CONTINUE,\
+Args=[--conf,spark.hadoop.fs.s3a.access.key=AKIAIOSFODNN7EXAMPLE,\
```
Comment (ozkatz) [inline docs/using/emr.md:121]: example values or variables? whatever you choose, it should probably be consistent across the guide..

**Review by ozkatz — CHANGES_REQUESTED** [review]
Looks good! just making sure we didn't reverse the meaning of the access key/access secret variables..

File: docs/using/emr.md (hunk 4 above)
Comment (itaiad200) [reply docs/using/emr.md:121]: Done.

**Review by ozkatz — APPROVED** [review]
LGTM, thanks!!

**Pull request merged**